sql/moz-fx-data-shared-prod/fenix_derived/firefox_android_clients_v1/query.sql

```diff
@@ -447,7 +558,59 @@ SELECT
     COALESCE(
       _previous.metadata.install_source__source_ping_datetime,
       _current.metadata.install_source__source_ping_datetime
-    ) AS install_source__source_ping_datetime
+    ) AS install_source__source_ping_datetime,
+    COALESCE(
+      _previous.metadata.install_source__source_ping_datetime,
+      _current.metadata.install_source__source_ping_datetime
+    ) AS install_source__source_ping_datetime,
+    COALESCE(
+      _previous.metadata.install_source__source_ping_datetime,
+      _current.metadata.install_source__source_ping_datetime
+    ) AS install_source__source_ping_datetime,
+    COALESCE(
+      _previous.metadata.install_source__source_ping_datetime,
+      _current.metadata.install_source__source_ping_datetime
+    ) AS install_source__source_ping_datetime,
+    COALESCE(
+      _previous.metadata.install_source__source_ping_datetime,
+      _current.metadata.install_source__source_ping_datetime
+    ) AS install_source__source_ping_datetime,
+    COALESCE(
+      _previous.metadata.install_source__source_ping_datetime,
+      _current.metadata.install_source__source_ping_datetime
+    ) AS install_source__source_ping_datetime,
+    COALESCE(
+      _previous.metadata.install_source__source_ping_datetime,
+      _current.metadata.install_source__source_ping_datetime
+    ) AS install_source__source_ping_datetime,
+    COALESCE(
+      _previous.metadata.install_source__source_ping_datetime,
+      _current.metadata.install_source__source_ping_datetime
+    ) AS install_source__source_ping_datetime,
```

Comment:
copy+paste error?

Reply:
yes, looks like it 😬 Thanks for catching this.